anta/tests/security.py

```diff
@@ -0,0 +1,345 @@
+"""
+Test functions related to the EOS various security settings
+"""
+from __future__ import annotations
+
+import logging
+from typing import Any, Dict, Optional, cast
+
+from anta.models import AntaTest, AntaTestCommand
+
+logger = logging.getLogger(__name__)
+
+
+class VerifySSHStatus(AntaTest):
+    """
+    Verifies if the SSHD agent is disabled in the default VRF.
+
+    Expected Results:
+        * success: The test will pass if the SSHD agent is disabled in the default VRF.
+        * failure: The test will fail if the SSHD agent is NOT disabled in the default VRF.
```

Comment:
wondering if this would not be worth adding a VRF keyword (where the default would be default) to be able to check for example management VRF (common use case I think)

Reply:
I was wondering the same too. eAPI returns an error when it's disabled in a specified VRF though. Let me check what we can do.

Reply:
so should we wait for an update here @carl-baillargeon or merge as it is?

Reply:
Merge as it for now. I will wait your PR for templates and modify accordingly. Thanks